modules/products/jira/variables.tf

```diff
@@ -56,8 +56,8 @@ variable "jira_configuration" {
   description = "Jira resource spec and chart version"
   type        = map(any)
   validation {
-    condition = (length(var.jira_configuration) == 6 &&
-    alltrue([for o in keys(var.jira_configuration) : contains(["helm_version", "cpu", "mem", "min_heap", "max_heap", "reserved_code_cache"], o)]))
+    condition = (length(var.jira_configuration) == 7 &&
+    alltrue([for o in keys(var.jira_configuration) : contains(["helm_version", "cpu", "mem", "min_heap", "max_heap", "reserved_code_cache", "license"], o)]))
```

Comment:
Can we make license a standalone variable? We're trying to move away from nested variables, and also license doesn't really fit into "Jira resource spec and chart version" category.

Reply:
Ignore the above comment. I just realised that all other products are doing the same...